server/src/collaborations/use-cases/filterByAcademyAndStatus/filter-by-academy-and-status.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { NotFoundException, BadRequestException } from '@nestjs/common';
+
+import { InMemoryCollaborationsRepository } from '@collaborations/test/in-memory/inMemoryCollaborationsRepository';
+import { CollaborationsStatus } from '@shared/constants';
+import { FilterByAcademyAndStatus } from '@collaborations/use-cases';
+
+describe('Find Collaboration by Academy id', () => {
```

Comment:
Em um dos test coloca mais de um id no `academy_id` e testa se NÃO foi possível encontrar por esse id.

Utilizar o `.not` do JEST

Ex:
```
    const collaboration2 = await inMemoryCollaborationRepository.create({
      collaboration_type_id: 'e802eade-11c6-4c94-9e0b-37dfdfd67c87',
      collaborator_id: '3ed27bd9-9a87-4f75-9a84-17e1949311d3',
      academy_id: ['bb324053-cfe0-49ad-9c24-1d6bd368555e', '37952c7d-5b92-47f4-befe-340e6ed7d1a5'],
      status: CollaborationsStatus.PENDING,
    });
```